src/components/Camera/style.css

```diff
@@ -36,42 +42,76 @@
   }
 }
 
+.fullScreenCenteredButton {
+  padding: 0;
+  margin-left: auto;
+  margin-right: auto;
+}
+
 .btn:extend(.btn, .btn-centered, .btn-primary) {
   @media (--small-viewport) {
-    padding: 0;
-    position: absolute;
-    bottom: 48px;
-    margin-left: auto;
-    margin-right: auto;
-    left: 0;
-    right: 0;
+    .fullScreenCenteredButton();
   }
-  .btnText {
-    @media (--small-viewport) {
-      display: none;
-    }
+
+  .livenessCamera & {
+    .fullScreenCenteredButton();
+  }
+}
+
+.recordingIndicator {
+  font-size: 11px;
+  text-align: center;
+
+  /* Red circle */
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 8px;
+    height: 8px;
+    margin-right: 5px;
+    background: #DC2A2A;
+    border-radius: 4px;
+    line-height: 120%;
   }
 }
 
+.fullScreenRoundedIcon() {
+  width: 60px;
+  height: 60px;
+  border: none;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-color: transparent;
+  box-shadow: none;
+}
+
+.fullScreenStartRecordingIcon() {
+  .fullScreenRoundedIcon();
+  background-image: url('./assets/start.svg');
+}
+
+.fullScreenStopRecordingIcon() {
+  .fullScreenRoundedIcon();
+  background-image: url('./assets/stop.svg');
+}
+
 .startRecording {
```

Comment:
isn't `.startRecording` only active for liveness, if so do you know why we need the small media query?